src/modules/dashboard/components/Dialogs/EscrowFundsDialog/EscrowFundsDialogForm.tsx

```diff
@@ -0,0 +1,426 @@
+          </div>
+        ),
+      })),
+    [],
+  );
+
+  // @TODO replace the mock with real data
+  const requiredFunds = useMemo(
+    () =>
+      requiredFundsMock.map((token) => ({
+        label: token.name,
+        value: token.id,
+        children: (
+          <FormSection appearance={{ border: 'top' }}>
+            <div className={styles.requiredFundsRow}>
+              <span>
+                {token.isPartial ? (
+                  <>
+                    <FormattedMessage
+                      {...MSG.partialFund}
+                      values={{ name: token.name }}
+                    />
+                    <span className={styles.tokenTotal}>
+                      {` (${formatMessage(MSG.total)} ${token.name} `}
+                      <Numeral
+                        className={styles.tokenNumeralTiny}
+                        unit={getTokenDecimalsWithFallback(token.decimals)}
+                        value={
+                          token.total?.[COLONY_TOTAL_BALANCE_DOMAIN_ID]
+                            .amount ?? 0
+                        }
+                      />
+                      )
+                    </span>
+                  </>
+                ) : (
+                  <FormattedMessage
+                    {...MSG.fullFund}
+                    values={{ name: token.name }}
+                  />
+                )}
+              </span>
+
+              <span className={styles.icon}>
+                <TokenIcon
+                  className={styles.tokenIcon}
+                  token={token}
+                  name={token.name || token.address}
+                />
+              </span>
+
+              <Numeral
+                unit={getTokenDecimalsWithFallback(token.decimals)}
+                value={token.balances[COLONY_TOTAL_BALANCE_DOMAIN_ID].amount}
+                className={styles.tokenNumeralNormal}
+              />
+              <span className={styles.symbolNormal}>{token.symbol}</span>
+            </div>
+          </FormSection>
+        ),
+      })),
+    [formatMessage],
+  );
+
+  return (
+    <>
+      <div className={styles.dialogContainer}>
+        <DialogSection appearance={{ theme: 'heading' }}>
+          <div>
+            <DomainDropdown
+              colony={colony}
+              name="motionDomainId"
+              currentDomainId={domainID}
+              renderActiveOptionFn={renderActiveDomainOption}
+              filterOptionsFn={filterDomains}
+              onDomainChange={handleMotionDomainChange}
+              showAllDomains={false}
+              showDescription={false}
+            />
+          </div>
+
+          <div className={styles.forceContainer}>
+            <div className={styles.margin}>
+              <Toggle
+                label={{ id: 'label.force' }}
+                name="force"
+                appearance={{ theme: 'danger' }}
+              />
+            </div>
+            <Tooltip
```

Comment:
You don't need to use `Tooltip` component, it's built in the `Toggle`, you just need to pass this prop `tooltipText`.